webapp/src/components/rhs/rhs_about_description.tsx

```diff
@@ -36,7 +36,15 @@ const RHSAboutDescription = (props: DescriptionProps) => {
 
     if (!editing) {
         return (
-            <RenderedDescription onClick={() => setEditing(true)}>
+            <RenderedDescription
+                onClick={(event: React.MouseEvent<HTMLElement>) => {
```

Comment:
Can technically be inferred, but doesn't hurt to have
```suggestion
                onClick={(event) => {
```

Reply:
Oh, I missed this yesterday. Good catch, this was me trying to make the linter recognize `nodeName` as a property in `target`, but then I realized that a cast was indeed needed. And of course I forgot to remove the type here! Thanks :)